formbricks/formbricks
# feat: lazy loading and endless scrolling for responses page

## What does this PR do?
If you have a lot of responses, as of now, the page would really swamp up with all of them with a super tiny nav scroll making it extremely difficult from a UX perspective.

This PR limits to 10 responses and then as soon as the user reaches the second last response of the 10, loads the next 10 and so on. They are, as before, sorted in descending order as per date-time!

Attached video with an example with 3 Responses at a time (for testing purposes):

https://github.com/formbricks/formbricks/assets/55556994/974c9836-57a5-4816-b479-56cd9c27ef6d


## Type of change

## How should this be tested?

Have more than 10 responses on a survey? Open the summary and then the responses tab to see this in effect!

## Checklist

### Required


### Appreciated
